src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        ArrayList arrayList = new ArrayList();
+    }
+}
```

Comment:
remove Main class from PR (pull request)
try `git rm --cached Main.java` to remove here but keep it locally

Reply:
Not fixed

Reply:
I don't know how to check that it's done but I get rm 'src/main/java/core/basesyntax/Main.java', so I hope it's done

Reply:
Still there, but it's not a big deal for now. Please, pay attention to this remark in your next PRs